tw-entrypoints/src/main/java/com/transferwise/common/entrypoints/tableaccessstatistics/TableAccessStatisticsSpyqlListener.java

```diff
@@ -120,18 +127,26 @@ protected ParsedQuery parseSql(String sql, TwContext context) {
       return new ParsedQuery();
     }
 
-    ParsedQuery result = new ParsedQuery();
-    long startTimeMs = System.currentTimeMillis();
+    var result = new ParsedQuery();
```

Comment:
Personally I don't see how using `var` here makes it better. `long` vs `var`, `ParsedQuery` vs `var`. In this case using `var` hurts the readability more in my opinion.